src/modules/fancyzones/editor/FancyZonesEditor/GridZone.xaml.cs

```diff
@@ -64,14 +64,14 @@ public GridZone()
             };
             Body.Children.Add(_splitter);
 
-            ((App)Application.Current).ZoneSettings[0].PropertyChanged += ZoneSettings_PropertyChanged;
+            App.ZoneSettings[MonitorVM.CurrentMonitor].PropertyChanged += ZoneSettings_PropertyChanged;
         }
 
         private void ZoneSettings_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
         {
             if (e.PropertyName == "IsShiftKeyPressed")
```

Comment:
Here you could use nameof(Settings.IsShiftKeyPressed) rather than have hardcoded strings

Reply:
Oh, that's a good point! I didn't write this code so I didn't really try to reformat things outside the area I was working with 🤔 ~ Not sure if I should...

Reply:
Oh right. It is never a bad thing to leave things better than when you found them :) But feel free to ignore this if you don't have spare time.